src/pages/Chat/Chat.tsx

```diff
@@ -98,11 +95,44 @@ const Chat: React.FC<ChatProps> = ({
     on(EVENTS.RECEIVE_MESSAGE, (data) => setAllMessages([...data]));
 
     chat.current = newChat;
+    chat.current.startMessageFetchProcess(topic);
+    console.info("Message fetch process started.");
+    chat.current.startUserFetchProcess(topic);
+    console.info("User fetch process started, interval: ", chat.current.getUserUpdateIntervalConst());
     setChatLoaded(true);
   };
 
+  const resendStuckMessages = async () => {
+    const beingSentThreshold = 1 * MINUTE;
+    const now = Date.now();
+    const messagesOlderThanThreshold = beingSentMessages.filter((msg) => msg.timestamp < now-beingSentThreshold);
+
+    for (let i = 0; i < messagesOlderThanThreshold.length; i++) {
+      const newlyConstructedMessage: MessageData = {
+        address: messagesOlderThanThreshold[i].address,
+        username: messagesOlderThanThreshold[i].username,
+        message: JSON.stringify({
+          text: messagesOlderThanThreshold[i].message,
+          threadId: messagesOlderThanThreshold[i].threadId,
+          messageId: messagesOlderThanThreshold[i].messageId,
+          parent: messagesOlderThanThreshold[i].parent
+        }),
+        timestamp: messagesOlderThanThreshold[i].timestamp
+      }
+      console.info("Resending message: ", newlyConstructedMessage)
+      const sResult = await chat.current?.sendMessage(ownAddress, topic, newlyConstructedMessage, stamp, privKey);
+      console.log("sResult ", sResult)
+    }
+  }
+
   useEffect(() => {
-    const messageIds = allMessages.map((msg) => JSON.parse(msg.message).messageId)
+    console.log("ALL MESSAGES: ", allMessages);
```

Comment:
is is necessary to log every message (allMessages) ?

Reply:
I removed it